docs/index.rst

```diff
@@ -30,8 +30,7 @@ the return type:
     async def foo(request: web.Request) -> APIResponse[list[str], Literal[200]]:
         return APIResponse(["foo"])
 
-Or for `Class Based Views
-<https://aiohttp.readthedocs.io/en/stable/web_quickstart.html#class-based-views>`
+Or for `Class Based Views <https://aiohttp.readthedocs.io/en/stable/web_quickstart.html#class-based-views>`_
```

Comment:
I wonder if this could be replaced with a Sphinx reference, since it's already mapped in intersphinx..